content/enterprise_influxdb/v1.9/administration/manage/security/authentication_and_authorization.md

```diff
@@ -25,70 +25,10 @@ This document covers setting up and managing authentication and authorization in
 - [HTTP Errors](#authentication-and-authorization-http-errors)
 
 ## Authentication
+### Enable Authentication
 
-Enable authentication in InfluxDB Enterprise
-to only allow requests that are sent with valid credentials to execute.
-
-{{% note %}}
-#### Plugins not authenticated
-Authentication only occurs at the HTTP request scope.
-Plugins do not currently have the ability to authenticate requests and service
-endpoints (for example, Graphite, collectd, etc.) are not authenticated.
-{{% /note %}}
-
-{{% note %}}
-#### Authentication recommended on public endpoints
-If InfluxDB Enterprise is being deployed on a publicly accessible endpoint,
-we **strongly recommend** enabling authentication.
-Otherwise, data and potentially destructive commands will be publicly available to any unauthenticated user.
-For additional security,
-InfluxDB Enterprise should be run behind a third-party service.
-Authentication and authorization should not be soley relied upon
-to prevent access and protect data from malicious actors.
-{{% /note %}}
-
-### Enable authentication
-
-Authentication is disabled by default in InfluxDB and InfluxDB Enterprise.
-All credentials are silently ignored, and all users have all privileges.
-
-To enable authentication in a cluster, do the following:
-
-1. **Create at least one [admin user](#admin-users)**.
-
-   To create an admin user,
-   run the following command using the [`influx` CLI](/enterprise_influxdb/v1.9/tools/influx-cli/):
-   ```
-   CREATE USER admin WITH PASSWORD 'mypassword' WITH ALL PRIVILEGES
-   ```
-
-2. **Enable authentication in your meta and data configuration files**.
-
-   Set the `auth-enabled` options to `true` in the `[http]` section:
-
-   ```toml
-   [http]
-     enabled = true
-     bind-address = ":8086"
-     auth-enabled = true # Set to true
-     log-enabled = true
-     write-tracing = false
-     pprof-enabled = true
-     pprof-auth-enabled = true
-     debug-pprof-enabled = false
-     ping-auth-enabled = true
-     https-enabled = true
-     https-certificate = "/etc/ssl/influxdb.pem"
-   ```
-
-   {{% note %}}
-If `pprof-enabled` is set to `true`, set `pprof-auth-enabled` and `ping-auth-enabled`
-to `true` to require authentication on profiling and ping endpoints.
-   {{% /note %}}
-
-3. **Restart InfluxDB Enterprise**.
-   Once restarted, InfluxDB Enterprise checks user credentials on every request and only
-   processes requests that have valid credentials for an existing user.
+For an introduction to authentication in InfluxDB Enterprise,
+see ["Get started with authentication and authorization"](/enterprise_influxdb/v1.9/administration/manage/security/get-started-auth).
```

Comment:
Authentication is actually a configuration step.  It isn't a "manage" step.  So recommend this move to configure per #3472